### ISSUE TITLE
error code

### ISSUE THREAD

**Issue by royalswe** [brillout/vite-plugin-ssr#904]
### Description

I stumbled on the error bellow and a Hydration node mismatch when implementing i18n.
The error will probably go away when im done implementing i18n but just in case if you want the error code.

```
useClientRouter.js?v=0fd9e360:131 Error: [vite-plugin-ssr@0.4.126][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error. (The error's stack trace is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at createErrorWithCleanStackTrace (createErrorWithCleanStackTrace.js?v=0fd9e360:8:15)
    at assert (assert.js?v=0fd9e360:32:27)
    at getExports.js?v=0fd9e360:34:13
    at Array.forEach (<anonymous>)
    at getExports (getExports.js?v=0fd9e360:31:37)
    at loadPageFilesClientSide (loadPageFilesClientSide.js?v=0fd9e360:26:73)
    at async getPageContextFirstRender (getPageContext.js?v=0fd9e360:35:39)
    at async fetchAndRender (useClientRouter.js?v=0fd9e360:124:35)
```

**Comment by brillout** [comment]
Thanks, I do indeed.

What's the `assert()` call that is failing at `getExports.js?v=0fd9e360:34:13`?

**Comment by royalswe** [comment]
path: /node_modules/vite-plugin-ssr/dist/esm/shared/getPageFiles/getExports.js?v=0fd9e360
![image](https://github.com/brillout/vite-plugin-ssr/assets/6486940/facc71c7-af58-4ce3-9741-11cbb643f396)

**Comment by brillout** [comment]
I see what's going on, let me fix this.

**Comment by brillout** [comment]
Fix released in `0.4.126-commit-340acd837`.
